src/makeCalculator.js

```diff
@@ -38,6 +38,41 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      calculator.result += num;
+
+      return calculator.result;
+    },
+    subtract(num) {
```

Comment:
```suggestion
    },
    
    subtract(num) {
```
Добавляй пробелы между методами

Reply:
Поняла! Спасибо!